icons/surveymonkey.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>SurveyMonkey icon</title><path d="M21.16271 13.1843a2.85799 2.85799 0 00-.67782.08405c-.84381-3.31811-3.54784-5.83763-6.91722-6.44522-.15087-.02694-.28665-.04742-.44074-.06681.02586-.82545.06358-1.77914 1.27805-2.63693l-.19182-.48385s-2.37076.73493-2.63908 2.75977c-.11746-.55066-1.22094-1.24034-1.76729-1.37073l-.27156.4386s.72416.36208.90089 1.36103c-3.36916.6056-6.07433 3.1229-6.92045 6.43983-1.51941-.37604-3.05597.55085-3.432 2.07026-.37604 1.5194.55085 3.05596 2.07026 3.432a2.8341 2.8341 0 001.71951-.11418 8.6889 8.6889 0 00.92028 1.71233l2.35244-1.58517-.02047-.02586c-.62179-.79959-1.00219-1.97635-1.06685-3.14772-.07005-1.29314.24462-2.5755 1.07007-3.32983 1.70156-1.44832 3.55613-.78774 4.71349.60023h.31143c1.16059-1.38797 3.01732-2.04747 4.71349-.60023.82438.75433 1.14012 2.03777 1.07007 3.32983-.06358 1.17137-.44398 2.34813-1.06684 3.14771l-.02048.02586 2.35244 1.58517a8.68902 8.68902 0 00.92244-1.71233c1.45514.57505 3.10095-.1384 3.676-1.59355.57505-1.45514-.1384-3.10095-1.59355-3.676a2.83324 2.83324 0 00-1.04469-.19826zM2.78612 16.84819c-.46184 0-.83623-.37439-.83623-.83623 0-.46184.37439-.83623.83623-.83623a.84371.84371 0 01.46876.1444c.00708.43912.03694.87758.08944 1.31361a.8363.8363 0 01-.56144.21984l.00324-.00539zm18.42724 0a.8362.8362 0 01-.5582-.21552 12.69894 12.69894 0 00.08944-1.31361c.38173-.25889.90106-.15931 1.15995.22242a.83513.83513 0 01.14396.47048c.00417.46063-.36587.83742-.8265.84159a.95763.95763 0 01-.00758.00003l-.00107-.00539z"/></svg>
```

Comment:
Hi @calvinespinoza, thanks for your work on this!
Is there any chance you could run this icon through SVGO(MG) to get the precision down to 3 decimals instead of 5? You can either visit [this site](https://jakearchibald.github.io/svgomg/) to complete it using a GUI, or from within your branch you can type `npm install` followed by `npm run svgo -- icons/surveymonkey.svg` to complete the optimization. Before submitting changes, it's also worth runnning the linter with `npm run svglint` to make sure the icon hasn't been messed up by the process!